package.json

```diff
@@ -75,7 +75,8 @@
     "@openmrs/esm-api": "^1.0.8",
     "@types/react-router": "^5.0.3",
     "react": "^16.9.0",
-    "react-dom": "^16.9.0"
+    "react-dom": "^16.9.0",
+    "timeago.js": "^4.0.1"
```

Comment:
I don't see the point of this library anymore, to format the dates, you can use this helper here https://github.com/openmrs/openmrs-esm-patient-chart/blob/75a0defb19477cda383011a32a64ac0e1a8e87ec/src/summary/documentation/dimension-helpers.ts#L10

Reply:
Good point. I will delete that.